eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/admin/handler/HTTPClientHandler.java

```diff
@@ -139,7 +139,7 @@ void list(HttpExchange httpExchange) throws IOException {
             });
 
             String result = JsonUtils.toJSONString(getClientResponseList);
-            httpExchange.sendResponseHeaders(200, result.getBytes().length);
+            httpExchange.sendResponseHeaders(200, result.getBytes(Constants.DEFAULT_CHARSET).length);
```

Comment:
Here `Constants.DEFAULT_CHARSET` needs to be imported:
org.apache.eventmesh.common.Constants;

Reply:
Thanks :)